src/components/Loading/Loading.component.tsx

```diff
@@ -1,9 +1,9 @@
 import { LoadingOutlined } from '@ant-design/icons';
-import React from 'react';
 import { Spin } from 'antd';
+import React, { FC } from 'react';
 
 const antIcon = <LoadingOutlined style={{ fontSize: 24 }} spin />;
 
-const Loading = () => <Spin aria-label='loading-spinner' indicator={antIcon} />;
+const Loading: FC = () => <Spin aria-label='loading-spinner' indicator={antIcon} />;
```

Comment:
FC should have at least one type. use FC<{}>

Reply:
Yup added!